eggdrop-basic.conf

```diff
@@ -65,7 +65,7 @@ set username "lamest"
 
 ## This setting defines which contact person should be shown in .status,
```

Comment:
Please revert this file to its original state, thanks

Reply:
`set admin "Lamer <email: lamer@lamest.lame.org>"` < here we use it as default setting, so you are fine with my change here?